mmf/configs/datasets/vqa2/with_raw_images.yaml

```diff
@@ -0,0 +1,75 @@
+dataset_config:
```

Comment:
There is lot of redundancy here which decreases readability. You would want to only override the stuff that needs to be overridden and skip common parts with defaults.

Reply:
when do we include defaults explicitly? I see this in some places and not others

```
includes:
- ./defaults.yaml
```

My guess is that we don't need to explicitly include it for mmf/configs/{models,datasets}/<x>/ but need to explicitly include it in mmf/projects/<x>/configs ?

Reply:
The defaults defined in `config_path` attribute are included automatically. Everything else needs to be included separately. In this case, since `with_raw_images.yaml` is an extension, it will need to be included separately.